### PR TITLE
Thymeleaf Crime template

### LINKED ISSUE #142: Create a Crime view page in Thymeleaf

### LINKED ISSUE #152: Add VictimService to Crime

### PR DESCRIPTION
**Pull request by FelixJacobsen**
Fix #142 

Adding a time does currently not work as intended. 

I have not examined this very carefully but the format seems to atleast be correct. 

image
image

A solution would be to change to LocalDate but then we can't display hh:ss which is crucial for a crime.

### CODE REVIEW

File: src/main/java/se/iths/crimedatabase/view/CrimeThymeleafController.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    @GetMapping("/deleteCrime")
+    public String deleteCrime(@RequestParam Long id) {
+        crimeService.delete(id);
+        return "redirect:/crimes";
+    }
+
+
+}
```
Comment (LordRekishi) [inline src/main/java/se/iths/crimedatabase/view/CrimeThymeleafController.java]:
```suggestion
}

```

File: src/main/resources/templates/add-crime-form.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        </div>
+
+        <div class="col-sm-8">
+            <button class="btn btn-primary" type="submit">Save</button>
+        </div>
+        <input th:field="*{id}" type="hidden"/>
+    </form>
+</div>
+</body>
+</html>
```
Comment (LordRekishi) [inline src/main/resources/templates/add-crime-form.html]:
```suggestion
</html>

```

File: src/main/resources/templates/list-crimes.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+    <script type="text/javascript" src="https://cdn.datatables.net/v/bs4/dt-1.11.5/datatables.min.js"></script>
+        <script>
+        $(document).ready(function(){
+        $("#category-table").DataTable();
+        })
+        </script>
+    </body>
+</html>
```
Comment (LordRekishi) [inline src/main/resources/templates/list-crimes.html]:
```suggestion
</html>

```

**Review by LordRekishi — CHANGES_REQUESTED** [review]

**Review by LordRekishi — APPROVED** [review]

**Comment by FelixJacobsen** [comment]
Fix #152

**Comment by FelixJacobsen** [comment]
Fix #152

**Pull request merged**